### PR TITLE
Random Beneficiary Names + Refactor

### PR DESCRIPTION
**Pull request by CalvinLe360**
closes: https://github.com/amlcloud/seeder/issues/67

### CODE REVIEW

File: lib/entity/generate_transactions_button.dart [hunk 1]
```diff
@@ -171,42 +172,32 @@ addPeriodicDataToList(
...
+}
+
 addRandomDataToList(List<Map<String, dynamic>> configList, String period,
     Jiffy dateIterator, int Counter) {
   List<Map<String, dynamic>> listData = [];
-  var random = math.Random();
+
   configList.forEach((configData) {
     configData[period].where((date) => date == Counter).forEach((config) {
-      DateTime configDate = DateTime(
```
Comment (ThuvaMathu) [inline lib/entity/generate_transactions_button.dart:191]: pull from main again. This code damage the algorithm.

File: lib/entity/generate_transactions_button.dart [hunk 2]
```diff
@@ -171,42 +172,32 @@ addPeriodicDataToList(
...
-          dateIterator.month,
-          dateIterator.date,
-          random.nextInt(24),
-          random.nextInt(60),
-          random.nextInt(3600),
-          random.nextInt(3600000));
-      double amount = configData['minAmount'] +
-          random.nextInt(configData['maxAmount'] - configData['minAmount']);
-      listData.add({
-        'amount': amount,
```
Comment (ThuvaMathu) [inline lib/entity/generate_transactions_button.dart:202]: Don't delete any code in generate transaction

Comment (CalvinLe360) [reply lib/entity/generate_transactions_button.dart:202]: this is a refactor because this exact code appeared twice

Comment (CalvinLe360) [reply lib/entity/generate_transactions_button.dart]: just pulled from main, i'll fix the implementation to adapt to it

**Pull request closed without merging**